collection-scripts/gather_priority_and_fairness

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# If the cluster version is < 4.5 skip this test
+VERSION=$(oc get clusterversion version -o jsonpath='{.status.desired.version}')
+COMPARE=$(echo -e "4.5\n${VERSION}" | sort -V | head -n1)
+if [[ ${COMPARE} != "4.5" ]]; then
+  exit 0
+fi
```

Comment:
@sferich888 I understand, the '!=' is not clear enough. 

The reason for != is simply because that's how it works when we use sort -V. It creates sorted list of versions.

I didn't find bash to be good at comparing decimal numbers. We could use bc? Whats your suggestions here to use instead of sort -V?

Would help if I add a comment to the script with the way how this works?

We set the current version to the VERSION variable:

VERSION=4.7
COMPARE=$(echo -e "4.5\n${VERSION}" | sort -V | head -n1)

echo $COMPARE
4.5

if version is >=4.5 then the COMPARE will always == 4.5

Or example if the version is lower:

VERSION=4.4
COMPARE=$(echo -e "4.5\n${VERSION}" | sort -V | head -n1)
echo $COMPARE
4.4

Then we skip the run.

Reply:
Commenting the code is always a good idea (it helps with maintenance). 
This is simply an interesting (and non-intuitive) way to do the comparison (and overcome the inadequacies with bash).

Reply:
@sferich888 I agree with you. Thank you. Added comments to explain the code bit better.